src/main/java/entities/Tag.java

```diff
@@ -0,0 +1,4 @@
+package entities;
+
+public class Tag {
```

Comment:
Maybe don't merge/Git ignore Tag, because someone else (@chazzybearr) has already implemented Tag on his branch and this can create a conflict when merging.

Reply:
It should be ok, the Tag entity here is just a placeholder, and the code isn't too complicated that it'll cause a merge conflict (its literally 2 lines)